lib/puppet/pops/lookup/hiera_config.rb

```diff
@@ -323,7 +365,15 @@ def create_configured_data_providers(lookup_invocation, parent_data_provider)
     data_providers = {}
 
     [@config[KEY_BACKENDS]].flatten.each do |backend|
-      raise Puppet::DataBinding::LookupError, "#{@config_path}: Backend '#{backend}' defined more than once" if data_providers.include?(backend)
+      if data_providers.include?(backend)
+        first_line = find_line_matching(/[^\w]#{backend}(?:[^\w]|$)/)
```

Comment:
Yes, unless we think that backends will be named using such characters. Given that backends are on their way out, I don't think we need to worry about that.